### PR TITLE
Add PandO (Python) to the list of RSE courses

### PR DESCRIPTION
**Pull request by Robadob**
Despite it being under Norbert's remit, it's developed/ran by RSE.

### CODE REVIEW

File: pages/training/index.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ The RSE Team focusses on training in version control as this is a key skill for
 
 - [git & GitHub through GitKraken - from Zero to Hero!](courses/git_Hero.md)
 - [Introduction to Deep Learning](courses/Intro_DL.md)
+- [Performance Profiling & Optimisation (Python)](/pando-python)
```
Comment (ns-rse) [inline pages/training/index.md:17]: Do we need to create this [page](https://rse.shef.ac.uk/pages/training/courses/pando-python) or is it meant to point to [myDevelopment](https://mydevelopment.csod.com/ui/lms-learning-details/app/event/fe34df6f-ab5f-49cf-b794-88aba03b3803)?

Comment (Robadob) [reply pages/training/index.md:17]: It should link to https://rse.shef.ac.uk/pando-python, the actual course.

The preceding slash makes it relative to site root, I tested it here: https://rse-mirror.robadob.org/training/

Comment (Robadob) [reply pages/training/index.md:17]: Annoyingly, you can't link directly to pages on MyDevelopment. They redirect to staff.sheffield.ac.uk when followed (e.g. yours above).

Comment (Robadob) [reply pages/training/index.md:17]: After that redirect, the link does work on second/third click :| Useless

Comment (ns-rse) [reply pages/training/index.md:17]: Ah ok, my mistunderstanding thanks :+1:

**Review by ns-rse — APPROVED** [review]
Simple and effective.

Perhaps we could consider having upcoming events individually listed under the [training](https://rse.shef.ac.uk/training/) page, although I appreciate it is often a duplication of effort to have the same things listed in multiple places.

**Comment by Robadob** [comment]
> Perhaps we could consider having upcoming events individually listed under the [training](https://rse.shef.ac.uk/training/) page, although I appreciate it is often a duplication of effort to have the same things listed in multiple places.

This perhaps requires discussion, as I would expect we should move all of ours courses over to MyDevelopment in the long-term, as it's now the central hub for course registration. (Just drives me nuts you can't have a clean link directly to a course's page, because of the redirects if you haven't got in-date cookies)

**Pull request merged**